corehq/apps/reports/tasks.py

```diff
@@ -59,28 +59,18 @@
     ReportNotification,
     UnsupportedScheduledReportError,
 )
-from .scheduled import get_scheduled_reports
+from .scheduled import get_scheduled_report_ids
 
 
 logging = get_task_logger(__name__)
 EXPIRE_TIME = 60 * 60 * 24
 
 
-def send_delayed_report(report):
+def send_delayed_report(report_id):
```

Comment:
thanks you're totally right. for now i'm going to leave because i'm just trying to keep my head above water with the interrupt queue